src/containers/NavContainer/NavContainer.scss

```diff
@@ -0,0 +1,39 @@
+.nav-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  height:100vh;
+  width: 21%;
```

Comment:
Use whole numbers (rounded to nearest 5) and use view width (vw) instead of percentages for reasons discussed.